packages/typespec-client-generator-core/test/methods/paged-operation.test.ts

```diff
@@ -805,6 +805,50 @@ it("next link with re-injected parameters", async () => {
   );
 });
 
+it("unbranded next link with re-injected parameters", async () => {
+  await runner.compileWithBuiltInAzureCoreService(`
```

Comment:
Maybe I misunderstand something. Wondering why the test is unbranded nextlink, but compile with builtin **Azure**CoreService?